go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/lightninglabs/lndmon
 
+go 1.19
```

Comment:
micro nit: leave the version field at the bottom of the file and move it accordingly in the `tools/go.mod`? I know this isn't super consistent across all projects but I'd guess the version at the end is more common (and I tend to move it down there whenever I touch a `go.mod` file)...